### PR TITLE
ACP2E-2290: [CLARIFICATION] Hidden sections in Pagebuilder returned in GraphQL response

### PR DESCRIPTION
**Pull request by thiaramus**
## Purpose of this pull request

This pull request (PR) is to add a note that CMS block content will contain HTML and CSS code and that elements hidden with CSS styles from the admin area and other system elements that are hidden during the rendering of the content by the browser or browser component, will be visible in the raw response.

## Affected pages

- https://developer.adobe.com/commerce/webapi/graphql/schema/store/queries/dynamic-blocks/
- https://developer.adobe.com/commerce/webapi/graphql/schema/store/queries/cms-blocks/

## Links to Magento Open Source code

- https://jira.corp.adobe.com/browse/ACP2E-2290

### CODE REVIEW

**Comment by thiaramus** [comment]
I am keeping this as a draft while working on the update to experience league in case I realize that I will need to change something here as well. Thanks.

**Comment by thiaramus** [comment]
@jeff-matthews, this one is ready for review :)
Thanks!

File: src/pages/graphql/schema/store/queries/cms-blocks.md [hunk 1]
```diff
@@ -52,6 +52,10 @@ The following query returns information about the `login-data` block:
 }
 ```
 
+<InlineAlert variant="info" slots="text" />
+
+Note that the `content` field may contain HTML markup and CSS styles. Hidden and system elements will be included in the response.
```
Comment (keharper) [inline src/pages/graphql/schema/store/queries/cms-blocks.md]:
```suggestion
The `content` field may contain HTML markup and CSS styles. Hidden and system elements may be included in the response.
```

File: src/pages/graphql/schema/store/queries/cms-page.md [hunk 2]
```diff
@@ -58,6 +58,10 @@ You must include the CMS page identifier value to retrieve the content of a spec
 }
 ```
 
+<InlineAlert variant="info" slots="text" />
+
+Note that the `content` field may contain HTML markup and CSS styles. Hidden and system elements will be included in the response.
```
Comment (keharper) [inline src/pages/graphql/schema/store/queries/cms-page.md:63]:
```suggestion
The `content` field may contain HTML markup and CSS styles. Hidden and system elements may be included in the response.
```

**Review by keharper — APPROVED** [review]

**Pull request merged**